sdk/conftest.py

```diff
@@ -26,33 +26,51 @@
 import os
 import pytest
 
+from devtools_testutils import recorded_test, test_proxy, variable_recorder
```

Comment:
We don't have azure-sdk-tools installed when we hit this point in CI. That's why we're seeing 

> unknown module

[here](https://dev.azure.com/azure-sdk/public/_build/results?buildId=1710032&view=logs&j=69247760-d00b-59ba-802c-4e283c193199&t=6654d4af-1717-52b7-cca2-b6c5ec5727ae&l=6244) I don't know how you haven't seen this before though.

Reply:
Hmm I see. Is that something we could/should change? My thinking with importing these fixtures in the repo-wide `conftest.py` is that you wouldn't have to import anything in order to record your tests. Having this import statement required in library-specific `conftest.py` files wouldn't be terrible by any means though

Reply:
Well, let me know if you can get it working with this in common conftest. I'm def not going to block you on it :)